aquadoggo/src/test_helpers.rs

```diff
@@ -162,3 +166,45 @@ pub fn shutdown_handle() -> JoinHandle<()> {
 
     shutdown
 }
+
+/// Create test database.
+pub async fn initialize_db() -> Pool {
```

Comment:
This looks a lot like the below method. Could it just call `initialize_db_with_url` and pass `TEST_CONFIG.database_url` as an argument?